gen/definitions/device_ha_pair.yaml

```diff
@@ -0,0 +1,156 @@
+    data_path: [ftdHABootstrap]
+    description: Select the encyption key generation scheme.
+    example: "CUSTOM"
+    enum_values: [ AUTO, CUSTOM ]
+  - model_name: standbyIP
+    type: String
+    mandatory: true
+    tf_name: lan_failover_standby_ip
+    data_path: [ftdHABootstrap,lanFailover]
+    example: "1.1.1.2"
+  - model_name: activeIP
+    tf_name: lan_failover_active_ip
+    mandatory: true
+    type: String
+    data_path: [ftdHABootstrap,lanFailover]
+    example: "1.1.1.1"
+  - model_name: logicalName
+    type: String
+    mandatory: true
+    tf_name: lan_failover_name
+    data_path: [ftdHABootstrap,lanFailover]
+    example: LAN-INTERFACE
+  - model_name: subnetMask
+    type: String
+    tf_name: lan_failover_subnet_mask
+    data_path: [ftdHABootstrap,lanFailover]
+    example: "255.255.255.0"
+  - model_name: useIPv6Address
+    type: Bool
+    tf_name: lan_failover_ipv6_addr
+    data_path: [ftdHABootstrap,lanFailover]
+    example: false
+  - model_name: name
+    type: String
+    tf_name: lan_failover_interface_name
+    data_path: [ftdHABootstrap,lanFailover,interfaceObject]
+    description: Name of physical interface
+    example: GigabitEthernet0/0
+  - model_name: id
+    type: String
+    mandatory: true
+    tf_name: lan_failover_interface_id
+    data_path: [ftdHABootstrap,lanFailover,interfaceObject]
+    description: ID of physical interface.
+    example: 757kdgh5-41c4-4558-a4d0-a8c07ac08470
+  - model_name: type
+    type: String
+    tf_name: lan_failover_interface_type
+    data_path: [ftdHABootstrap,lanFailover,interfaceObject]
+    description: Type of physical interface.
+    example: PhysicalInterface
+  - model_name: standbyIP
+    type: String
+    tf_name: stateful_failover_standby_ip
+    data_path: [ftdHABootstrap,statefulFailover]
+    example: "10.10.10.2"
+  - model_name: activeIP
+    tf_name: stateful_failover_active_ip
+    type: String
+    data_path: [ftdHABootstrap,statefulFailover]
+    example: "10.10.10.1"
+  - model_name: logicalName
+    type: String
+    tf_name: stateful_failover_name
+    data_path: [ftdHABootstrap,statefulFailover]
+    example: Stateful-INTERFACE
+  - model_name: subnetMask
+    type: String
+    tf_name: stateful_failover_subnet_mask
+    data_path: [ftdHABootstrap,statefulFailover]
+    example: "255.255.255.0"
+  - model_name: useIPv6Address
+    type: Bool
+    tf_name: stateful_failover_ipv6_addr
+    data_path: [ftdHABootstrap,statefulFailover]
+    example: false
+  - model_name: name
+    type: String
+    tf_name: stateful_failover_interface_name
+    data_path: [ftdHABootstrap,statefulFailover,interfaceObject]
+    description: Name of physical interface
+    example: GigabitEthernet0/0
+  - model_name: id
+    type: String
+    tf_name: stateful_failover_interface_id
+    data_path: [ftdHABootstrap,statefulFailover,interfaceObject]
+    description: ID of physical interface.
+    example: 76d24097-hj7r-7786-a4d0-a8c07ac08470
+  - model_name: type
```

Comment:
field is mandatory:
│ Failed to configure object (POST/PUT), got error: HTTP Request failed: StatusCode 400,
│ {"error":{"category":"FRAMEWORK","messages":[{"description":"LAN failover interface object type is missing.
│ for LAN failover link.\n\n"}],"severity":"ERROR"}}

Reply:
done